src/content/app-platform/defaulting-validation/index.md

```diff
@@ -109,6 +109,11 @@ Currently we validate:
 - All referenced configmaps and secrets exist.
 - The catalog has a matching AppCatalog CR.
 
+If app-operator finds a matching AppCatalogEntry CR named `{appCatalog}-{app}-{version}`, it will run more two validation.
+
+- Cloud provider compatibility (e.g. you can’t install kiam app in azure, for example)
+- Namespace restriction (cluster singleton, namespace singleton, fixed namespace) (e.g You can only install cert-manager in the kube-system namespace.)
```

Comment:
```suggestion
- Namespace restriction (cluster singleton, namespace singleton, fixed namespace) (e.g You can only install cert-manager in the kube-system namespace).
```

I think the full stop should be at the end. I think the usage of singleton so we don't need to explain that further.

Reply:
>  I think the usage of singleton so we don't need to explain that further. 

Sorry, I don't get this part, do you think we should remove the explanation part of this?

Reply:
My bad I meant to say.

I think the meaning of singleton is clear so we don't need to explain that further.